doc/lua_api.md

```diff
@@ -339,11 +339,11 @@ Naming conventions
 
 Registered names should generally be in this format:
 
-    modname:<whatever>
+    modname:<your_mod_name>
```

Comment:
This is wrong. The format is `<modname>:<whatever>` and not `<modname>:<modname>`.
e.g. `default:stone` and not `default:default`

Reply:
IMO, the whole part about naming conventions is confusing:

> Naming conventions
> ------------------
> 
> Registered names should generally be in this format:
> 
>     modname:<whatever>
> 
> `<whatever>` can have these characters:
> 
>     a-zA-Z0-9_

Registered name for what? What can go in `<whatever>`? 
But nevertheless thanks for your feedback. I really appreciate it.